src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -81,11 +117,76 @@ def severity_substitution(label, testcase, security_severity):
   if not data_types.SecuritySeverity.is_valid(security_severity):
     security_severity = data_types.SecuritySeverity.HIGH
 
-  security_severity_string = label_utils.severity_to_string(security_severity)
+  security_severity_string = severity_analyzer.severity_to_string(
+      security_severity)
   return [label.replace('%SEVERITY%', security_severity_string)]
 
 
-def apply_substitutions(label, testcase, security_severity):
+def impact_to_string(impact):
+  """Convert an impact value to a human-readable string."""
+  impact_map = {
+      data_types.SecurityImpact.STABLE: 'Stable',
+      data_types.SecurityImpact.BETA: 'Beta',
+      data_types.SecurityImpact.HEAD: 'Head',
+      data_types.SecurityImpact.NONE: 'None',
+      data_types.SecurityImpact.MISSING: data_types.MISSING_VALUE_STRING,
+  }
+
+  return impact_map[impact]
+
+
+def get_impact_from_labels(labels_lowercase):
```

Comment:
nit: maybe move the `.lower()` conversion into this function? That way it won't have any assumptions regarding the arguments, and the `labels_lowercase` isn't re-used anywhere anyway, just done once when calling this function on line 161.

Reply:
Uploading a PR for this!